src/Weather.java

```diff
@@ -0,0 +1,170 @@
+
+import components.map.Map;
+import components.map.Map1L;
+import components.simplewriter.SimpleWriter;
+import components.simplewriter.SimpleWriter1L;
+import components.utilities.FormatChecker;
+
+/**
+ * @author Haechan Jung
+ *
+ */
+public class Weather<date, condition> {
```

Comment:
I assume this doesn't compile. I don't think you even need the generic types here. Those will be hidden from the client.